lib/commands/exam.js

```diff
@@ -24,7 +24,8 @@ module.exports = TestCommand.extend({
     { name: 'load-balance',     type: Boolean, default: false, description: 'Load balance test modules. Test files will be sorted by weight from slowest (acceptance) to fastest (eslint).' },
     { name: 'random',           type: String,  default: false, description: 'Randomizes your modules and tests while running your test suite.' },
     { name: 'replay-execution', type: String,  default: false, description: 'A JSON file path which maps from browser number to a list of modules' },
-    { name: 'replay-browser',   type: [Array, Number, String], description: 'The number of the browser(s) to run from a specified file path' }
+    { name: 'replay-browser',   type: [Array, Number, String], description: 'The number of the browser(s) to run from a specified file path' },
+    { name: 'execution-only',   type: Boolean, default: false, description: 'Disables writing a test-execution json file after running your test suite'}
```

Comment:
I feel `execution-only` doesn't quite convey what the command actually do.
How about `--no-execution-file` and have an alias (short hand command) `--nef' ?

Reply:
sounds right :)